### PR TITLE
tmt, tmt-run, tmt-try: add Chinese translation

### PR DESCRIPTION
**Pull request by idorax**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 1.38.0**

### CODE REVIEW

**Comment by martinhoyer** [comment]
Nice, thanks @idorax!  
I'd suggest using tldr-lint, for example `npx tldr-lint ./tmt-try.md`

**Comment by spageektti** [comment]
@idorax could you please sign the CLA?

**Comment by idorax** [comment]
> @idorax could you please sign the CLA?

Sure. DONE :-)

File: pages.zh/linux/tmt-run.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# tmt run
+
+> 执行测试步骤。默认情况下，所有测试步骤都被执行。
+> 更多信息: <https://tmt.readthedocs.io/en/stable/overview.html#run>.
```
[the remark this answers is not in the document]
Comment (idorax) [reply pages.zh/linux/tmt-run.md]: Yes, colon in Chinese characters `：` should be used.
Thanks for review, it's updated.

File: pages.zh/linux/tmt-try.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# tmt try
+
+> 测试及环境快速上手。
+> 更多信息: <https://tmt.readthedocs.io/en/stable/stories/cli.html#try>.
```
[the remark this answers is not in the document]
Comment (idorax) [reply pages.zh/linux/tmt-try.md]: Updated.

File: pages.zh/linux/tmt.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# tmt
+
+> 创建、运行和调试测试的测试管理工具。
+> 诸如`运行`、`尝试`等子命令，均有相应的用法文档。
+> 更多信息: <https://tmt.readthedocs.io>.
```
[the remark this answers is not in the document]
Comment (idorax) [reply pages.zh/linux/tmt.md]: Updated.

**Comment by idorax** [comment]
> Nice, thanks @idorax! I'd suggest using tldr-lint, for example `npx tldr-lint ./tmt-try.md`
> 
> EDIT: nevermind, it all looks good to me. (I can't really review the translation :) ) Not sure why the bot thinks the 'more info' doesn't match.

Hi @martinhoyer, the issues are fixed :-)

**Comment by idorax** [comment]
Hi @blueskyson, @einverne, @spageektti and @martinhoyer, would you please help to review this PR? Thanks very much!

**Review by blueskyson — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**